gourmet/backends/db.py

```diff
@@ -894,28 +894,33 @@ def get_unique_values (self, colname,table=None,**criteria):
                   ]
         return [x for x in retval if x is not None] # Don't return null values
 
-    def get_ingkeys_with_count (self, search={}):
+    def get_ingkeys_with_count (self, search=None):
```

Comment:
We know that the signature is:
```python
def get_ingkeys_with_count(search: Optional[Dict] = None) -> List[Tuple[int, str]]: 
```

(since we're modifying this line, we may as well remove that blank space ;))

Reply:
Not only `Dict` but `Dict[str, Any]` and we need to replace that by an `Enum` sooner or later.

Reply:
I will change the function signature to:
def get_ingkeys_with_count(self, search: Optional[Mapping[str, Any]] = None) -> List[Tuple[int, str]]: